Muvr/MRAppDelegate.swift

```diff
@@ -73,6 +73,19 @@ class MRAppDelegate: UIResponder, UIApplicationDelegate, MKExerciseModelSource,
         return model
     }
     
+    func getSlackingModel() -> MKExerciseModel {
+        let bundlePath = NSBundle.mainBundle().pathForResource("Models", ofType: "bundle")!
+        
+        let modelPath = NSBundle(path: bundlePath)!.pathForResource("activity", ofType: "raw")!
```

Comment:
you need to copy the weight from `Classification.playground/Resources/activity.raw` to `Muvr/Models.bundle/activity.raw`